### DISCUSSION TITLE
How to use regular colorscheme in float-window agenda?

### DISCUSSION THREAD

**Discussion by Maltimore** [nvim-orgmode/orgmode#294] [Q&A] [answered]
I'm using the new floating window agenda view. It works well, but the colorscheme is rather different:

![image](https://user-images.githubusercontent.com/10359335/168134642-299b1651-ed33-4334-a8f7-5cdb950f15f8.png)

This also happens with a minimally changed `minimal_init.lua` in which the only option I added was to use the floating window for the agenda. How can I change the colorscheme of the agenda float to be my regular one?

**Comment by kristijanhusak** [comment, marked as answer]
This should make it work:
```lua
vim.api.nvim_create_autocmd('FileType', {
  pattern = {'orgagenda', 'org'},
  command = 'setlocal winhl=NormalFloat:Normal,FloatBorder:Normal'
})
```
This will use your regular background for both border and background color of the float window.
You can see how to change this in `:help 'winhl'`

**Reply by Maltimore** [reply]
Works like a charm! Thank you very much.
In the end, I incorporated this into the `win_split_mode` function from the docs:

```
  win_split_mode = function(name)
    local bufnr = vim.api.nvim_create_buf(false, true)
    --- Setting buffer name is required
    vim.api.nvim_buf_set_name(bufnr, name)
    local fill = 0.8
    local width = math.floor((vim.o.columns * fill))
    local height = math.floor((vim.o.lines * fill))
    local row = math.floor(((vim.o.lines - height) / 2))
    local col = math.floor(((vim.o.columns - width) / 2))
    win = vim.api.nvim_open_win(bufnr, true, {
      relative = "editor",
      width = width,
      height = height,
      row = row,
      col = col,
      style = "minimal",
      border = "rounded"
    })
    vim.api.nvim_win_set_option(win, 'winhighlight', 'NormalFloat:Normal,FloatBorder:Normal')
  end
```
See the last line, which is the only line that I changed compared to the docs.
